src/run.ts

```diff
@@ -10,14 +10,16 @@ import {
   getChangedPackages,
   sortTheThings,
   getVersionsByDirectory,
+  getReleaseMessage,
 } from "./utils";
 import * as gitUtils from "./gitUtils";
 import readChangesetState from "./readChangesetState";
 import resolveFrom from "resolve-from";
+import issueParser from "issue-parser";
```

Comment:
q: in the GitHub interface we have "Linked issues", I assume that this list is received by their frontend from their backend and that they don't have to prepare the PR body to get that. Would it be possible to leverage that? Or are there some limitations regarding this technique? One thing that comes to my mind - the list might include more than we want but maybe it's associated with some metadata that we could leverage to filter that list?

Reply:
I agree with you, I wish I had found a more appropriate solution but I think we don't have the necessary information in the API response (see https://docs.github.com/en/rest/reference/pulls#get-a-pull-request).

I tried with a Pull Request that has several related issues and they don't appear in the API response.

Reply:
There seems to be a way using the GraphQL API: https://github.community/t/get-all-issues-linked-to-a-pull-request/14653/6

Reply:
This doc also claims that it's possible to query that ingo using filters: https://docs.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests#search-for-linked-issues-and-pull-requests

But I couldn't make it work to get actual issue numbers.

Reply:
```graphql
fragment IssueOrPullRequestData on ReferencedSubject {
  ... on Issue {
    number
    body
    issueState: state
  }
  ... on PullRequest {
    number
    state
  }
}

{
  resource(url: "https://github.com/graphql/graphiql/pull/1914") {
    ... on PullRequest {
      timelineItems(itemTypes: [
        CONNECTED_EVENT, 
        DISCONNECTED_EVENT, 
        CROSS_REFERENCED_EVENT, 
        REFERENCED_EVENT
			], first: 100) {
        nodes {
          __typename
          ... on ReferencedEvent {
            id
            actor {
              url
            }
            isCrossRepository
            isDirectReference
            subject {
              ...IssueOrPullRequestData
            }
          }
          ... on DisconnectedEvent {
            id
            subject {
              ...IssueOrPullRequestData
            }
          }
          ... on CrossReferencedEvent {
            resourcePath
            isCrossRepository
            willCloseTarget
            target {
              ...IssueOrPullRequestData
            }
            source {
              ...IssueOrPullRequestData
            }
          }
        }
      }
    }
  }
}
```

got a little closer, but not quite. using this query, I was able to get the PR referenced from a referenced issue, but not the referenced issue itself :laughing: ...

Reply:
This can now be queried with:
```graphql
{
  resource(url: "https://github.com/graphql/graphiql/pull/1914") {
    ... on PullRequest {
      closingIssuesReferences(first: 99) {
        nodes {
          number
        }
      }
    }
  }
}
```

I will be revamping this PR this week to hopefully land it before the end of the year.